Source/CommandBar/labelPosition.js

```diff
@@ -0,0 +1,11 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+const labelPosition = {
```

Comment:
Great to see the JavaScript convention being used properly :) lower camelCase for non instantiateable types.